stock_landed_costs_auto_create/models/stock_picking.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def action_done(self):
+        res = super().action_done()
+        for item in self.filtered(
+            lambda x: x.state == "done"
+            and x.picking_type_code == "incoming"
+            and x.purchase_id
+            and x.purchase_id.landed_costs_ids
```

Comment:
I think this is equivalent:
```suggestion
            and x.purchase_id.landed_costs_ids
```